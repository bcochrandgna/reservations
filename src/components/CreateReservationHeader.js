import React from 'react';
import { View, Text } from 'react-native';
import i18n from '../helpers/i18n';
import styles from '../styles/components/CreateReservationHeader';

const CreateReservationHeader = () => (
  <View style={styles.container}>
    <Text style={styles.titleText}>{i18n.t('create_reservation')}</Text>
  </View>
);

export default CreateReservationHeader;
