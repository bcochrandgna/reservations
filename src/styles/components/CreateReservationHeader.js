import { StyleSheet } from 'react-native';
import { standardPadding, largeTextSize, headingTextColor } from '..';

export default StyleSheet.create({
  container: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    alignItems: 'center',
    padding: standardPadding,
  },
  titleText: {
    fontSize: largeTextSize,
    color: headingTextColor,
  },
});
