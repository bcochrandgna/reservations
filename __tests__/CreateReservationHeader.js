import React from 'react';
import { shallow } from 'enzyme';
import CreateReservationHeader from '../src/components/CreateReservationHeader';

describe('CreateReservationHeader', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<CreateReservationHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
