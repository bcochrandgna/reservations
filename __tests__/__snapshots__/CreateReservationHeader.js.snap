// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateReservationHeader renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "justifyContent": "space-between",
      "padding": 5,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#007AFF",
        "fontSize": 18,
      }
    }
  >
    Create Reservation
  </Text>
</View>
`;
